dashboards/aws.ebs.summary.json

```diff
@@ -31,7 +31,7 @@
         "properties": {
           "elementScopeTypes": "[\"EBS\"]",
           "metricLimit": "10",
-          "metrics": "[{\"fqn\":\"netuitive.aws.ebs.totalops\",\"aggFn\":\"sum\"},{\"fqn\":\"netuitive.aws.ebs.totalbytespersec\",\"aggFn\":\"sum\"}]",
+          "metrics": "[{\"fqn\":\"aws.ebs.volumequeuelength\",\"aggFn\":\"sum\"},{\"fqn\":\"netuitive.aws.ebs.queuelengthdifferential\",\"aggFn\":\"sum\"}]",
```

Comment:
netuitive.aws.ebs.queuelengthdifferential should use the avg aggFn instead of sum

Reply:
Fixed